.github/workflows/issues.yml

```
@@ -7,6 +7,9 @@ on:
 env:
   IN_MAINTAINER_LIST: ${{ contains(secrets.MAINTAINER_LIST, github.actor) }}
 
+permissions:
+  contents: read
+
```

This workflow uses a custom token, and so we should disable all scopes.

```suggestion
permissions: {}
```